compiler/src/dotty/tools/dotc/Driver.scala

```diff
@@ -26,7 +26,7 @@ class Driver extends DotClass {
     if (fileNames.nonEmpty)
       try {
         val run = compiler.newRun
-        run.compile(fileNames)
+        run.compile(fileNames.distinct)
```

Comment:
Not sure this is a great thing to do:
- It doesn't deduplicate between `Foo.scala` and `/a/b/c/Foo.scala`
- List#distinct is O(n^2), that's not great when compiling 10000 files.

Reply:
Then maybe I should keep only the fix on the TASTY ones. With those the compiler crashes. With `dotc Foo.scala Foo.scala` the compiler will emit the error saying that the class is duplicated, which is already fine.

The implementation of `List@distinct` I see is in `SeqLike` and it does look `O(n)`.

Reply:
Ah indeed, List#distinct uses a hashset so it's probably more like n*log(n) which is acceptable. But why does the compiler crash when compiling twice the same TASTY file? Could we get it to output an error instead?